includes/classes/Comments/Comments.php

```diff
@@ -0,0 +1,222 @@
+<?php
+/**
+ * Disable Comments
+ *
+ * @package 10up-experience
+ */
+
+namespace TenUpExperience\Comments;
+
+use TenUpExperience\Singleton;
+
+/**
+ * Comments class
+ */
+class Comments {
+
+	use Singleton;
+
+	/**
+	 * Setup module
+	 *
+	 * @since 1.11.2
+	 */
+	public function setup() {
+		if ( TENUP_EXPERIENCE_IS_NETWORK ) {
+			add_action( 'wpmu_options', [ $this, 'disable_comments_settings' ] );
+			add_action( 'admin_init', [ $this, 'save_disable_comments_settings' ] );
+		} else {
+			add_action( 'admin_init', [ $this, 'single_site_setting' ] );
+		}
+
+		if ( 'no' === $this->get_setting() ) {
+			return;
+		}
```

Comment:
**thought (non-blocking):** We want to be wary that we don't break any sites that currently support comments. What does this default to? We should default to comments enabled.

Reply:
@darylldoyle Yeah, the default value for disabling comments is `no`, so it won't break any sites that currently support comments :)